pkg/portal/graphql/collaborator.go

```diff
@@ -8,8 +8,36 @@ var collaborator = graphql.NewObject(graphql.ObjectConfig{
 	Name:        "Collaborator",
 	Description: "Collaborator of an app",
 	Fields: graphql.Fields{
-		"id":        &graphql.Field{Type: graphql.NewNonNull(graphql.String)},
-		"userID":    &graphql.Field{Type: graphql.NewNonNull(graphql.String)},
+		"id": &graphql.Field{Type: graphql.NewNonNull(graphql.String)},
+
+		// AppID is intentionally excluded because
```

Comment:
Circular reference between types in GraphQL schema should be okay. I think the problem is Go does not allow circular dependency between package variable initializer? Can we use `init()` function to initialize them instead?

ref: https://github.com/graphql-go/graphql/issues/164

Reply:
It is not very useful to add `app`. Do we really need this?

Reply:
Not really... I just saw the comment and provide a possible workaround.